apps/daimo-web/package.json

```diff
@@ -28,6 +28,7 @@
     "pino-pretty": "^10.2.3",
     "react": "18.2.0",
     "react-dom": "^18.2.0",
+    "sharp": "^0.33.2",
```

Comment:
👍 fixes console warnings?

Reply:
Actually, it's for the image resizing. I tried to get imagescript to work, but my efforts were abortive.